nebula-cache-manager/core/classes/CacheManager.cls

```diff
@@ -367,6 +379,15 @@ public without sharing class CacheManager {
       return this.keyToValue.get(key);
     }
 
+    public Object get(String key, System.Type cacheBuilderClass) {
+      if (!this.contains(key)) {
```

Comment:
```suggestion
      if (this.contains(key) == false) {
```

I'll save the bulk of the review for @jongpie but we both really value consistency across the codebase when it comes to comparisons.

I also _love_ single returns in methods but I'm not pure about it, personally; here, I'd be tempted to have another return statement after the `this.put(key, value);` call to optimize performance. It's not as clean, but it also seems to align with the `get` implementation for `CacheManager.PlatformCache`.